more/free-programming-playgrounds.md

```diff
@@ -62,25 +63,33 @@
 * [Educational Visual CPU Simulator](https://github.com/Belotti01/CPU-Visual-Simulator) - Renato Cortinovis, Nicola Preda, Jonathan Cancelli, Alessandro Belotti, Davide Riva (JAVA, JAR)
 
 
+### <a id="bash"></a>Bash/Shell
+
+* [Bash Crawl](https://gitlab.com/slackermedia/bashcrawl)
+* [Terminal Quest](https://github.com/KanoComputing/terminal-quest)
```

Comment:
Playgrounds are the ones which dont need any installation. Please remove any that require it

Reply:
to bad we dont have programs or games category for this.